src/Function.cpp

```diff
@@ -562,6 +570,12 @@ Function::OutputForwardDecl(std::ostream &out)
 	OutputHeader(out);
 	if(func_attr_inline)
 		out << " __attribute__((always_inline))";
+	if(func_attr_aligned){
+		int value = 1;
+		int power = rnd_upto(16);
+		value = value << power;
+		out << " __attribute__((aligned(" <<value << ")))";
```

Comment:
Thank you. One last nit: replace it with `<< (1 << power) <<` and add space after '<<' in order to make coding style consistent.

Reply:
Done